modules/admin_manual/pages/installation/manual_installation/upgrade_install_phpmyadmin.adoc

```diff
@@ -27,77 +27,159 @@ sudo apt-cache policy phpmyadmin
-correct location.
+To download `phpmyadmin`, see the {phpmyadmin_dl_url}[phpmyadmin download page] and select the latest stable version or a version that fits your needs. Then start downloading, extracting and putting the files to the correct location.
 
-=== Download and Move
+==== Download and Move
 
-The following example uses `phpMyAdmin-5.0.4-all-languages` as basis for upgrading. It renames the
-old installation because we need to keep some basic configuration settings.
+The following example uses `phpMyAdmin-5.2.0-all-languages` as the example for upgrading. The old installation location will be renamed, because we need to keep some basic configuration settings.
 
 [source,bash]
 ----
 cd /tmp
-sudo wget https://files.phpmyadmin.net/phpMyAdmin/5.0.4/phpMyAdmin-5.0.4-all-languages.tar.gz
-sudo tar -xvf phpMyAdmin-5.0.4-all-languages.tar.gz
+----
+
+[source,bash]
+----
+sudo wget https://files.phpmyadmin.net/phpMyAdmin/5.2.0/phpMyAdmin-5.2.0-all-languages.tar.gz
+----
+
+[source,bash]
+----
+sudo tar -xvf phpMyAdmin-5.2.0-all-languages.tar.gz
+----
+
+[source,bash]
+----
 sudo mv /usr/share/phpmyadmin /usr/share/phpmyadmin.old
+----
+
+[source,bash]
+----
 sudo mkdir -p /usr/share/phpmyadmin
-sudo cp -r phpMyAdmin-5.0.4-all-languages/. /usr/share/phpmyadmin
-sudo rm -r phpMyAdmin-5.0.4-all-languages*
 ----
 
-=== Recreate Basic Configuration Settings
+[source,bash]
+----
+sudo cp -r phpMyAdmin-5.2.0-all-languages/. \
+      /usr/share/phpmyadmin
+----
+
+[source,bash]
+----
+sudo rm -r phpMyAdmin-5.2.0-all-languages*
+----
+
+==== Recreate Basic Configuration Settings
 
-Because the upgrade has default basic configuration settings, it is necessary to recreate them
-based on the former settings coming from the Ubuntu installation. The file to change can be found
-in `/usr/share/phpmyadmin/libraries/vendor_config.php`. To keep a copy of the original file from
-the update, create a backup with:
+Because the upgrade has some default basic configuration settings, it is necessary to recreate them based on the former settings coming from the Ubuntu installation. The file to change can be found in `/usr/share/phpmyadmin/libraries/vendor_config.php`. +
+To keep a copy of the original file from the download, create a backup with:
 
 [source,bash]
 ----
-cd /usr/share/
-sudo cp phpmyadmin/libraries/vendor_config.php phpmyadmin/libraries/vendor_config.php.orig
+sudo cp /usr/share/phpmyadmin/libraries/vendor_config.php \
+        /usr/share/phpmyadmin/libraries/vendor_config.php.orig
 ----
 
-Then, open two editors of your choice in parallel with following files:
+NOTE: The transformation of values may differ depending on the phpmyadmin version compared. Especially starting with version 5.2.0, the definitions have changed from constants to array. In addition, keys have the same name but different casing and the definition of `CONFIG_DIR`, which defines the path to the configuration files has been removed completely and needs to be set manually.
```

Comment:
Not sure what "different casing" means. If this refers to upper and lower cases, better use "capitalization" instead.